requirements.txt

```diff
@@ -9,7 +9,7 @@ Flask-Login==0.5.0
 Flask-SQLAlchemy==2.5.1
 Flask-WTF==0.14.3
 greenlet==1.0.0
-idna==3.1
+idna==2.8
```

Comment:
did you manually change this? if so, why? Why has it been changed to an older version?